pkg/infra/iac2/templates/s3_bucket_policy/factory.ts

```diff
@@ -0,0 +1,15 @@
+import * as aws from '@pulumi/aws'
+
+interface Args {
+    Name: string
+    Bucket: aws.s3.Bucket
+    Policy: aws.iam.PolicyDocument
+}
+
+// noinspection JSUnusedLocalSymbols
+function create(args: Args): aws.s3.BucketPolicy {
+    return new aws.s3.BucketPolicy(args.Name, {
+        bucket: args.Bucket.id, // refer to the bucket created earlier
```

Comment:
What is this comment trying to hint at, and who is it for —  us, or the user? The user will see this.

Reply:
let me remove that, was probably in deploylib and i missed removing it